### DISCUSSION TITLE
Conditional collections

### DISCUSSION THREAD

**Discussion by rodrigopina360** [ppy/osu#28356] [Ideas]
Mainly just an idea I had that could turn out to be interesting.

In short, have the ability to create collection patterns based on conditions introduced by the user. For example, creating a collection with the condition "status=r,l bpm<220 stars>6" and have all maps that meet the criteria show up. It's not really a new concept, given search patterns already exist, but my main proposal would be a way to store those search patterns (probably more conveniently as a collection) for some more complex queries I've seen users make (and made myself).

I'm not really sure if it's something that's been requested before, or if it's even an idea worth considering, but in the off chance I'm the first person to suggest such an idea I'll just leave it here.

**Comment by AkiraTenchi** [comment]
I would like to second this. 

Having a text file for often used queries feels quite impractical so having something like that would be a huge QoL improvement.

**Comment by shiumano** [comment]
https://github.com/ppy/osu/issues/22204
How about asking for additional search presets in this song selection wedge redesign?
I imagine it would be like a mod preset.

**Comment by k3tzchen** [comment]
Maybe also the ability to toggle specific mods for a given collection. Like using the filters `status=r,l` and having DT automatically enabled for the `Leaderboard [DT]` Collection
